test_runner/fixtures/neon_fixtures.py

```diff
@@ -1642,6 +1642,7 @@ def start(
             self.id, overrides=overrides, extra_env_vars=extra_env_vars
         )
         self.running = True
+        self.http_client().check_ready()
```

Comment:
Let's introduce a new parameter, by default `True` and use it like:

```suggestion
        if wait_tenants_loaded:
            self.http_client().wait_tenants_loaded()
```